linera-base/src/crypto.rs

```diff
@@ -463,9 +463,9 @@ impl Signature {
     {
         let mut msg = Vec::new();
         value.write(&mut msg);
-        let mut messages: Vec<&[u8]> = Vec::new();
-        let mut signatures: Vec<dalek::Signature> = Vec::new();
-        let mut public_keys: Vec<dalek::PublicKey> = Vec::new();
+        let mut messages = Vec::<&[u8]>::new();
```

Comment:
Try removing this type annotation and use `msg.as_slice()` below instead of `&msg`

Reply:
Correct, it works.